documentation/API-design-guidelines.md

```diff
@@ -949,15 +949,37 @@ The following controls will be performed on the access token:
 
 The scopes allow defining the permission scopes that a system or a user has on a resource, ensuring that they can only access the parts they need and not have access to more. These restrictions are done by limiting the permissions that are granted to OAuth tokens.
 
-Scopes should be represented as:
-- API Name: address-management, numbering-information...
-- Protected Resource: orders, billings…
+Scopes should be represented as below for all Camara APIs except the APIs that offer explicit event subscriptions:
+- API Name: qod, address-management, numbering-information...
+- Protected Resource: sessions, orders, billings…
 - Grant-level, action on resource: read, write…
+
+For e.g. qod:sessions:read
+
+The APIs that offer explicit event subscriptions must have a way to reflect which event types are being subscribed to, when a subscription create request is made. This will impact how consent management is handled for these APIs. 
+
+Scopes should be represented as below for APIs that offer explicit event subscriptions with action read and delete:
+ 
+- API Name: device-roaming-subscriptions
+- Protected Resource: subscriptions (If the API name offering explicit subscriptions does not include the word subscriptions, then the protected resource "subscriptions" should be added when defining the scope.)
+- Grant-level, action on resource: read, delete
+
+For e.g. device-roaming-subscriptions:read 
+  
+Scopes should be represented as below for APIs that offer explicit event subscriptions with action create:
+
```

Comment:
Should we explicitly write that we are defining here a scope for each event-type in order to manage subscribed consent at this level for subscription creation?

Reply:
done